src/redux/features/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from "@reduxjs/toolkit";
+const initialState = []
+
```

Comment:
- [ ] Dear @DjibrilM, I noticed you have a folder called `feature` which holds both the `bookSlice.js` and ` categoriesSlice.js` files. This is against the `ducks pattern`. To make sure the use of the `ducks pattern` please create a separate folder for both files and put the files in their respective folders. 
 >  Something like `/src/redux/books/booksSlice.js` and `/src/redux/categories/categoriesSlice.js`